samples/s02-instances/deviceCode.hlsl

```diff
@@ -57,29 +57,39 @@ GPRT_RAYGEN_PROGRAM(simpleRayGen, (RayGenData, record))
   );
 
   const int fbOfs = pixelID.x + record.fbSize.x * pixelID.y;
-  gprt::store(record.fbPtr, fbOfs, gprt::make_rgba(payload.color));
+  gprt::store(record.fbPtr, fbOfs, gprt::make_bgra(payload.color));
```

Comment:
GPUs natively only really support BGRA framebuffers. Intel arc for example does not support RGBA framebuffers, only BGRA ones. BGRA is the only format compatible with all cards.

normally, OpenGL handles this sort of swizzling operation for us. But since we’re using Vulkan, Vulkan expects us to do the swizzle ourselves, since there is a bit of a performance tax doing this swizzle.